extensions/resource-deployment/src/services/tools/kubeCtlTool.ts

```diff
@@ -75,86 +75,93 @@ export class KubeCtlTool extends ToolBase {
 		command: 'apt-get update'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Kubectl.AptGetPackages', "getting packages needed for kubectl installation ..."),
+		comment: localize('resourceDeployment.Kubectl.AptGetPackages', "getting packages needed for kubectl installation …"),
 		command: 'apt-get install -y apt-transport-https'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Kubectl.DownloadAndInstallingSigningKey', "downloading and installing the signing key for kubectl ..."),
+		comment: localize('resourceDeployment.Kubectl.DownloadAndInstallingSigningKey', "downloading and installing the signing key for kubectl …"),
 		command: 'curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Kubectl.AddingKubectlRepositoryInformation', "adding the kubectl repository information ..."),
+		comment: localize('resourceDeployment.Kubectl.AddingKubectlRepositoryInformation', "adding the kubectl repository information …"),
 		command: 'echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Kubectl.AptGetUpdate', "updating repository information ..."),
+		comment: localize('resourceDeployment.Kubectl.AptGetUpdate', "updating repository information …"),
 		command: 'apt-get update'
 	},
 	{
 		sudo: true,
-		comment: localize('resourceDeployment.Kubectl.InstallingKubectl', "installing kubectl ..."),
+		comment: localize('resourceDeployment.Kubectl.InstallingKubectl', "installing kubectl …"),
 		command: 'apt-get install -y kubectl'
 	}
 ];
 // TODO: Remove dependency on curl on Win32 and use powershell Invoke-WebRequest instead
 const win32InstallationCommands = [
 	{
-		comment: localize('resourceDeployment.Kubectl.DeletePreviousDownloadedKubectl.exe', "deleting previously downloaded kubectl.exe if one exists ..."),
-		command: `IF EXIST .\kubectl.exe DEL /F .\kubectl.exe`,
+		comment: localize('resourceDeployment.Kubectl.DeletePreviousDownloadedKubectl.exe', "deleting previously downloaded kubectl.exe if one exists …"),
+		command: 'IF EXIST .\\kubectl.exe DEL /F .\\kubectl.exe',
 	},
 	{
-		comment: localize('resourceDeployment.Kubectl.DownloadingAndInstallingKubectl', "downloading and installing the latest kubectl.exe ..."),
-		command: `for /f %i in ('curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt') do curl -LO https://storage.googleapis.com/kubernetes-release/release/%i/bin/windows/amd64/kubectl.exe`
+		comment: localize('resourceDeployment.Kubectl.DownloadingAndInstallingKubectl', "downloading and installing the latest kubectl.exe …"),
+		command: `powershell -Command "& {(New-Object System.Net.WebClient).DownloadFile('https://storage.googleapis.com/kubernetes-release/release/v1.16.2/bin/windows/amd64/kubectl.exe', 'kubectl.exe')}"`
```

Comment:
Hm, this isn't really going to necessarily be true. If we do truly want to always get the latest then the suggestion on their page is to read [https://storage.googleapis.com/kubernetes-release/release/stable.txt](https://storage.googleapis.com/kubernetes-release/release/stable.txt) to find the latest stable version

Reply:
Oh yeah you already did this with the old curl version. So I'd fix this to do the same thing

---
In reply to: [339315886](https://github.com/microsoft/azuredatastudio/pull/8039#discussion_r339315886) [](ancestors = 339315886)

Reply:
No - Fixing the version was a conscious independent choice by Alan as we wanted it to be predictable.

Reply:
I will change code install version v1.16.2 across all platforms.

Reply:
no further investigation since brew install on mac and apt-get install on linux are pulling latest version, so I am changing to pull the latest version of kubectl for windows as well.

Reply:
Fixed to make new powershell implementation of what the curl based one was before.